data/Store.js

```diff
@@ -726,10 +729,9 @@ export class Store extends HoistBase {
     //------------------------
     // Private Implementation
     //------------------------
-
-    getOrThrow(id) {
+    getOrThrow(id, errorMsg) {
         const ret = this.getById(id);
-        throwIf(!ret, `Could not find record with id '${id}'`);
+        throwIf(!ret, `Could not find record with id '${id}'. ${errorMsg}`);
```

Comment:
Don -- think this is going to output "undefined" at the end of the string.

Reply:
Yikes. Right. I had that same thought and meant to test but got distracted. Thanks for catching will look tomorrow AM.